my_list.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'my_enum'
+
+class MyList
+  include MyEnumerable
+
+  def initialize(list)
+    @list = list
+  end
+
+  def each(&block)
+    @list.each(&block)
+  end
+end
```

Comment:
Kindly ensure your enumerable is well created by running `ruby -r "./my_list.rb" -e "p MyList.new(1,2,4)"` 👍🏽